spec/knapsack_pro/runners/queue/rspec_runner_spec.rb

```diff
@@ -45,7 +46,18 @@
             status: :next,
             runner: runner,
             can_initialize_queue: true,
-            args: ['--example-arg', 'example-value', '--format', 'progress', '--format', 'KnapsackPro::Formatters::RSpecQueueSummaryFormatter', '--default-path', 'fake-test-dir'],
+            args: [
+              '--example-arg',
+              'example-value',
+              '--format',
+              'progress',
+              '--format',
+              'KnapsackPro::Formatters::RSpecQueueSummaryFormatter',
+              '--format',
+              'KnapsackPro::Formatters::TimeTracker',
+              '--default-path',
+              'fake-test-dir',
```

Comment:
```suggestion
              '--example-arg', 'example-value',
              '--format', 'progress',
              '--format', 'KnapsackPro::Formatters::RSpecQueueSummaryFormatter',
              '--format', 'KnapsackPro::Formatters::TimeTracker',
              '--default-path', 'fake-test-dir',
```

It would be easier to read.